src/pkg/task/dao/execution.go

```diff
@@ -342,3 +349,22 @@ func (e *executionDAO) querySetter(ctx context.Context, query *q.Query) (orm.Que
 
 	return qs, nil
 }
+
+// Param keys is strings.Split() after trim "extra_attrs."/"ExtraAttrs." prefix
+func buildInClauseSqlForExtraAttrs(keys []string) string {
+	switch len(keys) {
+	case 0:
+		// won't fall into this case, as the if condition on "keyPrefix == key"
+		return fmt.Sprintf("select id from execution where extra_attrs=?")
```

Comment:
Remove these because it will never happen.

Reply:
> Remove these because it will never happen.
@heww 
we should make sure "default" case equivalent to "len(keys) >= 2",  therefore, this `case 0` should probably be kept.

Reply:
But the return string for case 0 is a bad result.

Reply:
> But the return string for case 0 is a bad result or unusable result.

so shall we `return ""` for `case 0:`

Reply:
because anyway `default` should be equivalent to `len(keys) >= 2`, I guess `case 0` should be listed/kept as a place holder.

Reply:
returns empty string is ok for me

Reply:
what's the sql result on the case 0 path? The executions with the extra_attrs is empty?

If you'd like to handle case 0 path, just add a if condition 

```
keys := strings.Split(strings.TrimPrefix(key, keyPrefix), ".")
if len(keys) == 0 {
    return qs, nil
}
```

Reply:
the case 0 of `buildInClauseSqlForExtraAttrs()` is just a place holder to ensure `default` case equivalent to "len(keys) >= 2". it won't fall into this `case 0` because there is a if check  condition`keyPrefix == key`.